instagram/spec/factories/post.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :post do
+    caption "nofilter"
+    image Rack::Test::UploadedFile.new(Rails.root + 'spec/images/test.jpg', 'image/jpg')
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)